### DISCUSSION TITLE
Hello, I need OITS to verify my research about tidal inversion,but I haven't received a reply yet

### DISCUSSION THREAD

**Discussion by gagelinxu** [pyTMD/pyTMD#86] [General]
Can you help me?  Thank you.

**Comment by tsutterley** [comment]
Sorry I must have missed it.  Can you explain what you're trying to do and where you're getting stuck?

**Reply by gagelinxu** [reply]
I found it is easy to use  TPXO9_atlas in TMD. I used to try to transform .nc files to binary file,but I don't know what I have missed out and always didn't get it work. IF possible, would do you  please share TPXO9-atlas **binary** files with me? Thanks a lot.

**Reply by tsutterley** [reply]
I am not the creator of the TPXO9 models and am _only_ licensed to use the outputs.  Registration information can be found on the TPXO website:
https://www.tpxo.net/tpxo-products-and-registration
Hope this helps
